resources/provider/migrations/postgres/24_v0.10.16.up.sql

```diff
@@ -1,6 +1,57 @@
 -- Autogenerated by migration tool on 2022-04-04 17:12:37
 -- CHANGEME: Verify or edit this file before proceeding
 
+-- Resource: codepipeline.pipelines
+CREATE TABLE IF NOT EXISTS "aws_codepipeline_pipelines" (
+                                                            "cq_id" uuid NOT NULL,
```

Comment:
nit: weird formatting. Did you reformat it manually like that from the output of migration tool?

Reply:
migration tool but I will check again

Reply:
Looks like GoLand's weird SQL formatting to me

Reply:
Yes, probably
Fixed

Reply:
Looks like it's back?